gradle/libs.versions.toml

```diff
@@ -25,7 +25,7 @@ dependencyAnalysisPlugin = "2.5.0"
 detektPlugin = "1.23.7"
 gitSemverPlugin = "0.12.11"
 jibPlugin = "3.4.4"
-kotlinPlugin = "2.0.21"
+kotlinPlugin = "2.1.0"
```

Comment:
Nit: We should have merged https://github.com/eclipse-apoapsis/ort-server/pull/1517 instead; for this PR Renovate created the wrong title, this is not about Kotlin serialization.